### PR TITLE
10 suspect progression bars

### PR DESCRIPTION
**Pull request by HarrisonShort**
- Adds suspect progression (indicator) bars
- Below gif tested by now removed test code to trigger indicator progression
suspect_indicators

### CODE REVIEW

File: Assets/Scenes/MainMenu.unity [hunk 1]
```diff
@@ -947,11 +947,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-<<<<<<< HEAD
```
Comment (njramage) [inline Assets/Scenes/MainMenu.unity]: One conflict on MainMenu scene

**Review by njramage — APPROVED** [review]

File: Assets/Scenes/MainMenu.unity (hunk 1 above)
Comment (LeLionDev) [reply Assets/Scenes/MainMenu.unity]: Heads up, I accepted the HEAD in my PR (#12)

File: Assets/Scripts/Suspect.cs [hunk 2]
```diff
@@ -17,6 +36,45 @@ public void SelectSuspect()
...
+        }
+
+        if (feature != null && !featureMatched && feature == suspectData.Feature)
+        {
+            featureMatched = true;
+            TurnOnNextIndicator();
+        }
+    }
+
+    private void TurnOnNextIndicator()
```
Comment (LeLionDev) [inline Assets/Scripts/Suspect.cs:69]: Just wanted to check with people if we wanted the progress bar as such, or were we wanting to do light up on specific matched item?

File: Assets/Scenes/MainMenu.unity (hunk 1 above)
Comment (njramage) [reply Assets/Scenes/MainMenu.unity]: My comment is wrong on this. I was reading the removal as a conflict

**Review by LeLionDev — CHANGES_REQUESTED** [review]

**Review by LeLionDev — APPROVED** [review]

**Pull request merged**